Library/Styles/ButtonStyles.swift

```diff
@@ -81,6 +92,13 @@ public let greenButtonStyle =
   <> UIButton.lens.layer.borderColor .~ UIColor.ksr_green_700.cgColor
   <> UIButton.lens.layer.borderWidth .~ 1.0
 
+public let roundedGreenButtonStyle: ButtonStyle = { button in
```

Comment:
We should eventually replace `greenButtonStyle` with the style from `roundedGreenButtonStyle` (because all buttons will be rounded in the future. For now, kept them separate.

Reply:
Noted in our upcoming button sweep.